hack/dockerfiles/docs.Dockerfile

```
@@ -14,6 +14,7 @@ RUN apk add --no-cache rsync git
 WORKDIR /src
 COPY --from=docsgen /out/docsgen /usr/bin
 ARG FORMATS
+ARG BUILDX_EXPERIMENTAL=0
```

Is it to not generate experimental flags for docs? Not sure how docs validation is ok with this as it should not set `--invoke` and `--print` flags in markdown table. Maybe I miss smth.

---

I've overwritten the flag in `docker-bake.hcl`, I think that should be source of truth for the config? I guess it would be a good idea to remove the default here then, I think I just wanted to set it to a "definitely-falsey" value.

---

Oh I see, then:

```suggestion
ARG BUILDX_EXPERIMENTAL
```

should be enough